opm/output/eclipse/EclFilesComparator.cpp

```diff
@@ -0,0 +1,127 @@
+/*
+   Copyright 2016 Statoil ASA.
+
+   This file is part of the Open Porous Media project (OPM).
+
+   OPM is free software: you can redistribute it and/or modify it under the terms of the GNU General Public License as published by
+   the Free Software Foundation, either version 3 of the License, or
+   (at your option) any later version.
+
+   OPM is distributed in the hope that it will be useful,
+   but WITHOUT ANY WARRANTY; without even the implied warranty of
+   MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+   GNU General Public License for more details.
+
+   You should have received a copy of the GNU General Public License
+   along with OPM.  If not, see <http://www.gnu.org/licenses/>.
+   */
+
+#include <opm/output/eclipse/EclFilesComparator.hpp>
+
+ECLFilesComparator::ECLFilesComparator(const char* unrstFile1, const char* unrstFile2) {
+    ecl_file1 = ecl_file_open(unrstFile1, ECL_FILE_CLOSE_STREAM);
+    ecl_file2 = ecl_file_open(unrstFile2, ECL_FILE_CLOSE_STREAM);
```

Comment:
The  `ECL_FILE_CLOSE_STREAM` will close the `FILE *` between every keyword access, this is a weird construction to limit use of filedescriptors (mainly on windows) - no reason to use this flag; just pass `0`.